ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -159,6 +159,9 @@ function RJSF(props) {
           ObjectFieldTemplate={CustomObjFieldTemplate}
           additionalMetaSchemas={[JS4]}
           uiSchema={uiSchema(jsonSchema)}
+          liveValidate
+          showErrorList={false}
```

Comment:
okay so liveValidation is live now. ErrorList is bad for forms now? should we show it on long run? like a red badge that lists out all the error if someone clicks on it.

Reply:
yes. That's the plan

Reply:
Goooooood